source/_docs/configuration.markdown

```diff
@@ -26,6 +26,10 @@ _If you use {% term "Home Assistant Operating System" %}, you can find `configur
 
 _If you use {% term "Home Assistant Core" %} , you can find `configuration.yaml` in the config folder passed to the `hass` command (default is `~/.homeassistant`)._
 
+### Do not store private data in your configuration.yaml file
+
+Avoid storing private data (passwords, API keys, etc.) directly in your `configuration.yaml` file. Private data can be stored in either a [separate file](/docs/configuration/secrets/) or in [environmental variables](/docs/configuration/yaml/#using-environment-variables), which circumvents this security problem.
```

Comment:
Which security problem?

Reply:
Oh, good question. I copied this over... probably this is referring to the fact that you share your information by accident when sharing configs. Which is not mentioned here. Let me rephrase...